packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -334,18 +334,23 @@ export abstract class SfdxCommandletExecutor<T>
       execution,
       cancellationToken
     );
-    CancellableStatusBar.show(execution, cancellationTokenSource);
+    await ProgressNotification.show(execution, cancellationTokenSource);
+    // CancellableStatusBar.show(execution, cancellationTokenSource); TODO: Remove this line when ProgressNotification is stable
```

Comment:
Nick suggested commenting out the CancellableStatusBar for now, until we can be certain that the new progress notification works as expected.